core/storageproviders/ipfsStorage.go

```diff
@@ -38,27 +42,32 @@ type IPFSStorage struct {
 	gateway       string
 }
 
-func (s *IPFSStorage) Setup(config Config) {
+//Setup sets up the ipfs storage for saving the video to ipfs
+func (s *IPFSStorage) Setup() error {
 	log.Println("Setting up IPFS for external storage of video. Please wait..")
 
-	s.gateway = config.IPFS.Gateway
+	s.gateway = ownconfig.Config.IPFS.Gateway
```

Comment:
I know in other places we're using the default `config.Config`, and from pure aesthetic reasons something like this `ownconfig.Config` looks nicer.  I guess we could do something also like `config.Current` or something too.

Reply:
I use the `config.Config` convention pretty often, so I hadn't thought twice about it. But I do agree with you here, the `ownconfig.Config` makes sense. The reason I switched in this file is because it already imported another `config` package. 🤔 

I'm conflicted on which one I like better as what you suggested makes sense, `config.Current`